staging/src/k8s.io/client-go/restmapper/category_expansion.go

```diff
@@ -58,7 +58,7 @@ func NewDiscoveryCategoryExpander(client discovery.DiscoveryInterface) CategoryE
 // Expand fulfills CategoryExpander
 func (e discoveryCategoryExpander) Expand(category string) ([]schema.GroupResource, bool) {
 	// Get all supported resources for groups and versions from server, if no resource found, fallback anyway.
-	apiResourceLists, _ := e.discoveryClient.ServerResources()
+	_, apiResourceLists, _ := e.discoveryClient.ServerGroupsAndResources()
```

Comment:
Why is this change needed?

Reply:
Because the static check said that ServerResources is a deprecated function

Reply:
Got it. Thanks for the explanation.